### PR TITLE
[CC-14622] ac doc documentation display discounted price on promotional product

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by jeremyfourna — APPROVED** [review]
OK for me

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md [hunk 1]
```diff
@@ -26,37 +26,37 @@ To start working with discounts, go to **Merchandising** > **Discount**.
...
+    2. Select the **DISCOUNT APPLICATION TYPE** and define which products are eligible for the discount. For reference information, in the **Create new discount** reference, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
  * Click **Next**, or select the **Conditions** tab to proceed.
- * On the *Create Discount* page, in the *Conditions* tab, do the following:
-    1. Select the *APPLY WHEN* conditions or click **Plain query** and enter the query manually. For reference information, in the Creating a voucher guide, see the [Conditions](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#conditions) section.
-    2. Enter the value for *THE DISCOUNT CAN BE APPLIED IF THE QUERY APPLIES FOR AT LEAST X ITEM(S).* field.
-* Click **Save** to create the new discount.
+ * On the **Conditions** tab, do the following:
```
Comment (lenadoc) [inline docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md:55]:
```suggestion
 * In the **Conditions** tab, do the following:
```

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md [hunk 2]
```diff
@@ -26,37 +26,37 @@ To start working with discounts, go to **Merchandising** > **Discount**.
...
+    1. Select the **APPLY WHEN** conditions or click **Plain query** and enter the query manually. For reference information, in the Creating a voucher guide, see the [Conditions](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#conditions) section.
```
Comment (lenadoc) [inline docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md:56]:
```suggestion
    1. Select the **APPLY WHEN** conditions or click **Plain query** and enter the query manually. For reference information, see [Conditions](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#conditions).
```

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md [hunk 3]
```diff
@@ -28,18 +28,18 @@ Review the reference information before you start, or just look up the necessary
 ## Creating a voucher
 
 To create a discount voucher:
-1. On the *Discount* page,  in the top-right corner, click **Create new discount**.
-2. On the *Create new discount* page, in the *General* tab, do the following:
-    1. In *STORE RELATION*, select the stores where you wish the discount to be active.
-    2. In the *DISCOUNT TYPE* drop-down, select **Voucher codes**.
+1. On the **Discount** page,  in the top-right corner, click **Create new discount**.
+2. On the **Create new discount** page, on the **General Information** tab, do the following:
```
Comment (lenadoc) [inline docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md:32]:
```suggestion
2. On the **Create new discount** page, in the **General Information** tab, do the following:
```

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md [hunk 4]
```diff
@@ -28,18 +28,18 @@ Review the reference information before you start, or just look up the necessary
...
-    5. _Optional_: in the **PRIORITY** field, enter an integer value from 1 to 9999 for the discount voucher priority. For reference information, see [General information tab](#general-information-tab).
+    4. _Optional_: in the **DESCRIPTION** field, enter the description of the voucher.
+    5. _Optional_: in the **PRIORITY** field, enter an integer value from `1` to `9999` for the discount voucher priority. For reference information, see [General Information tab](#general-information-tab).
     6. Specify if the voucher is exclusive. For reference information, see [General information tab](#general-information-tab).
     7. Specify the validity interval (lifetime) of the voucher.
- 1. Click **Next** or select the **Discount calculation** tab to proceed.
- 2. In the *Discount calculation* tab, do the following:
-    1.  In the **CALCULATOR TYPE** drop-down, select either *Percentage* or *Fixed amount*. For reference information, see [Discount calculation tab](#discount-calculation-tab).
+3. To proceed, click **Next** or select the **Discount calculation** tab.
+4. On the **Discount calculation** tab, do the following:
```
Comment (lenadoc) [inline docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md:41]:
```suggestion
4. In the **Discount calculation** tab, do the following:
```

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md [hunk 5]
```diff
@@ -52,21 +52,21 @@ To create a discount voucher:
       {% endinfo_block %}
 
     2. Select the **Discount application type** and define the products to which the voucher should be applied. See reference information of the [Discount calculation](#discount-calculation-tab) tab for more details.
- 3. Click **Next**, or select the *Conditions* tab to proceed.
- 4. On the *Create new discount* page, in the *Conditions* tab, do the following:
-    1. Select the **Apply when** conditions or click **Plain query** and enter the  query manually. See reference information of the [Conditions](#conditions) tab for more details.
-    2. Enter the value for the *THE DISCOUNT CAN BE APPLIED IF THE QUERY APPLIES FOR AT LEAST X ITEM(S).* field.
-3. Click **Save** to create the new voucher.
+5. Click **Next**, or select the **Conditions** tab to proceed.
+6. On the **Conditions** tab, do the following:
```
Comment (lenadoc) [inline docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md:56]:
```suggestion
6. In the **Conditions** tab, do the following:
```

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md [hunk 6]
```diff
@@ -52,21 +52,21 @@ To create a discount voucher:
...
+When you click **Save**, an additional tab named **Voucher Codes** appears. Here, you can generate, view, and export voucher codes (if they were already created).
 The list is empty until codes are generated.
 
-In the *Voucher code* tab, do the following:
-1. Enter the *QUANTITY* for voucher codes you want to generate.
-2. _Optional_: Enter a *CUSTOM CODE*.
-3. Set the *ADD RANDOM GENERATED CODE LENGTH* by selecting the value from a drop-down list.
-4. Set *MAX NUMBER OF USES*.
-5. Click **Generate** to complete the process.
+On the **Voucher code** tab, do the following:
```
Comment (lenadoc) [inline docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md:64]:
```suggestion
In the **Voucher code** tab, do the following:
```

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md [hunk 7]
```diff
@@ -78,7 +78,8 @@ See [Voucher code](#voucher-code) for more information.
 {% endinfo_block %}
 
 **Tips & tricks**
-<br>Once you generated voucher codes, you can export them as a .csv file.
+
+Once you generated voucher codes, you can export them as a .csv file.
```
Comment (lenadoc) [inline docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md:82]:
```suggestion
Once you generated voucher codes, you can export them as a CSV file.
```
https://developers.google.com/style/filenames?hl=en#file-type-names

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md [hunk 8]
```diff
@@ -125,7 +127,7 @@ The following table describes the attributes you enter and select in the *Genera
...
 | NON-EXCLUSIVE | Defines the discount exclusivity. Buyers can redeem non-exclusive discounts in conjunction with other non-exclusive discounts.|
 | EXCLUSIVE | Defines the discount exclusivity. When a discount is exclusive, no other discounts may be applied in conjunction. When a cart is eligible for multiple exclusive discounts, the discount with the highest value to the customer is applied. The exception to this is promotional product discounts. Query string discounts and promotional product discounts exclude only among each other. Promotional product discounts are not affected by exclusive query string discounts, and vice versa.|
-| VALID FROM and VALID TO| Vouchers are redeemable/the cart rule is active between Valid From and Valid To dates, inclusive. E.g., a voucher can be redeemed/discount applies to the cart starting from 1/1/2018 until 31/12/2019.|
+| VALID FROM and VALID TO| Vouchers are redeemable/the cart rule is active between Valid From and Valid To dates, inclusive. For example, a voucher can be redeemed/discount applies to the cart starting from 1/1/2018 until 31/12/2019.|
```
Comment (lenadoc) [inline docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md:130]:
```suggestion
| VALID FROM and VALID TO| Vouchers are redeemable or the cart rule is active between _Valid From_ and _Valid To_ dates, inclusive. For example, a voucher can be redeemed/discount applies to the cart starting from 1/1/2018 until 31/12/2019.|
```

https://developers.google.com/style/slashes?hl=en#slashes-with-alternatives

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/managing-discounts.md [hunk 9]
```diff
@@ -21,35 +21,35 @@ To manage discounts, navigate to **Merchandising** > **Discount**.
...
 
 To edit a discount, do the following:
-1. On the *Overview of Discounts* page, click **Edit** in the _Actions_ column.
+1. On the **Overview of Discounts** page, click **Edit** in the **Actions** column.
 2. Update the attributes and click **Save**.
 
 {% info_block infoBox %}
 
-To find out more information about the attributes that you see on the *Edit Discount* page, in the Creating a voucher guides, see the [Reference information: Creating a voucher](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#reference-information-creating-a-voucher) section.
+To find out more information about the attributes that you see on the **Edit Discount** page, in the Creating a voucher guides, see the [Reference information: Creating a voucher](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#reference-information-creating-a-voucher) section.
```
Comment (lenadoc) [inline docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/managing-discounts.md:29]:
```suggestion
For more information about the attributes on the **Edit Discount** page, see [Reference information: Creating a voucher](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#reference-information-creating-a-voucher).
```

File: docs/scos/user/features/202200.0/promotions-discounts-feature-overview.md [hunk 10]
```diff
@@ -136,8 +143,12 @@ The product selection based on the query string is dynamic:
 
 Promotional product is a discount application type that discounts a particular product at a set quantity, enabling "buy X, get Y" promotions.
 
-When a customer meets the conditions for a promotional product discount, a **Promotional products** section is displayed in the cart, allowing customers to add the available quantity of the discounted product to the cart.
-![Promotional product](https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Promotions+&+Discounts/Discount/Discount+Feature+Overview/promotional-product-frontend.png)
+When a customer meets the conditions for a promotional product discount, a **Promotional products** section is displayed in the cart and lets customers add the available quantity of the discounted product. The section consists of the product name, SKU, original and discounted price, and discount description.
```
Comment (lenadoc) [inline docs/scos/user/features/202200.0/promotions-discounts-feature-overview.md:146]:
```suggestion
When a customer meets the conditions for a promotional product discount, the _Promotional products_ section is displayed in the cart and lets customers add the available quantity of the discounted product. The section consists of the product name, SKU, original and discounted price, and discount description.
```

**Review by lenadoc — CHANGES_REQUESTED** [review]

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md [hunk 11]
```diff
@@ -26,37 +26,37 @@ To start working with discounts, go to **Merchandising** > **Discount**.
...
+    6. Specify if the discount is exclusive. For reference information, in the **Create new discount** reference, see [General information tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#general-information-tab).
     7. Specify the validity interval (lifetime) of the discount.
-* Click **Next** or select the *Discount calculation* tab to proceed.
+* Click **Next** or select the **Discount calculation** tab to proceed.
 
-* On the *Create Discount* page, in the *Discount calculation* tab, do the following:
-    1.  In the *CALCULATOR TYPE* drop-down, select either *Percentage* or *Fixed amount*. For reference information, in the *Create new discount* reference, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
+* On the **Discount calculation** tab, do the following:
```
[the remark this answers is not in the document]
Comment (VadymSachenko) [reply docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md:40]: According to https://developers.google.com/style/ui-elements, the _on_ preposition should be used here.

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md (hunk 1 above)
Comment (VadymSachenko) [reply docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md:55]: According to https://developers.google.com/style/ui-elements, the _on_ preposition should be used here.

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md (hunk 4 above)
Comment (VadymSachenko) [reply docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md:41]: According to https://developers.google.com/style/ui-elements, the _on_ preposition should be used here.

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md (hunk 6 above)
Comment (VadymSachenko) [reply docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md:64]: According to https://developers.google.com/style/ui-elements, the _on_ preposition should be used here.

File: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md (hunk 3 above)
Comment (VadymSachenko) [reply docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md:32]: According to https://developers.google.com/style/ui-elements, the _on_ preposition should be used here.

**Pull request merged**